### PR TITLE
Move share links, update movie url (#9334)

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

Three small changes to unfuck landing page. Move the share link to the bottom, add the JS for the Facebok share, and point to a different landing page for the movie.

## Issue / Bugzilla link

#9334 

## Testing

http://localhost:8000/en-US/firefox/unfck/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/campaign/unfck/index.html [hunk 1]
```diff
@@ -166,6 +144,28 @@ <h2 class="c-section-title">The Checklist</h2>
...
+      alt='Click the unfck button.',
+      include_cta=True,
+      id='share',
+      src_name='button'
+    ) %}
+      <p class="c-item-cta">
+        <a id="js-tw" class="mzp-c-cta-link" href="{{ tw }}" target="_blank" rel="external noopener" data-cta-type="link" data-cta-text="Checklist: share">Tweet this</a></p>
+      </p>
+      <p class="c-item-cta">
+        <a id="js-fb" class="mzp-c-cta-link" href="{{ fb }}" target="_blank" rel="external noopener" data-cta-type="link" data-cta-text="Checklist: share">Facebook this</a></p>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/campaign/unfck/index.html:165]: Not a blocker but just calling out. FaceBook Container (which the landing page promotes) gives a warning on this link: 
image

File: media/js/firefox/campaign/unfck.js [hunk 2]
```diff
@@ -28,33 +28,39 @@
...
+    function openFacebookSubwin(url) {
+        open(url, 'fb_share', 'height=380,width=660,resizable=0,toolbar=0,menubar=0,status=0,location=0,scrollbars=0').focus();
+    }
 
     function handleShareLinkClick(e) {
-        var href = e.target.href;
+        var link_href = e.target.href;
+        var service = '';
+
+        if(link_href.includes('twitter')) {
```
Comment (alexgibson) [inline media/js/firefox/campaign/unfck.js:39]: It looks like `includes` isn't supported by IE, so you may want to change to `indexOf()`?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

**Review by alexgibson — APPROVED** [review]
One suggestion for the JS, but otherwise this looks good to me. r+wc

File: bedrock/firefox/templates/firefox/campaign/unfck/index.html (hunk 1 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/campaign/unfck/index.html:165]: I don't think there's any way to link to Facebook without this happening.

**Pull request merged**